cluster/etcd_service_discovery.go

```diff
@@ -644,14 +651,18 @@ func (sd *etcdServiceDiscovery) watchEtcdChanges() {
 			case wResp, ok := <-chn:
 				if wResp.Err() != nil {
 					logger.Log.Warnf("etcd watcher response error: %s", wResp.Err())
-					time.Sleep(100 * time.Millisecond)
+					time.Sleep(10 * time.Millisecond)
```

Comment:
what is the rational for this change?

Reply:
I looked at the runtimes of the syncservers() in the new pitaya 1.1.8 servers we already have in production, and the function sometimes takes less than 10ms to run, so putting 100ms seemed a little excessive since we want the watcher to start processing the events again as soon as possible

Reply:
Actually, my mistake, this wasn't the line where the change from 100 to 10 ms was intended. fixing back to 100ms here